### ISSUE TITLE
GATConv with segment fault

### ISSUE THREAD

**Issue by kaisadadi** [pyg-team/pytorch_geometric#868]
## ❓ Questions & Help

I run the benchmark/citation/gat.py, and I get segment fault, what's wrong? 
What I get is as follows, can somebody help me?

Fatal Python error: Segmentation fault

Current thread 0x00007f75f9d81740 (most recent call first):
  File "/home/wke18/anaconda3/envs/IJC/lib/python3.7/site-packages/torch_scatter/max.py", line 13 in forward
  File "/home/wke18/anaconda3/envs/IJC/lib/python3.7/site-packages/torch_scatter/max.py", line 109 in scatter_max
  File "/home/wke18/anaconda3/envs/IJC/lib/python3.7/site-packages/torch_geometric/utils/softmax.py", line 23 in softmax
  File "/home/wke18/anaconda3/envs/IJC/lib/python3.7/site-packages/torch_geometric/nn/conv/gat_conv.py", line 103 in message
  File "/home/wke18/anaconda3/envs/IJC/lib/python3.7/site-packages/torch_geometric/nn/conv/message_passing.py", line 126 in propagate
  File "/home/wke18/anaconda3/envs/IJC/lib/python3.7/site-packages/torch_geometric/nn/conv/gat_conv.py", line 91 in forward
  File "/home/wke18/anaconda3/envs/IJC/lib/python3.7/site-packages/torch/nn/modules/module.py", line 547 in __call__
  File "utils/cora.py", line 71 in forward
  File "/home/wke18/anaconda3/envs/IJC/lib/python3.7/site-packages/torch/nn/modules/module.py", line 547 in __call__
  File "./utils/runcora.py", line 104 in train
  File "./utils/runcora.py", line 66 in run
  File "utils/cora.py", line 80 in <module>
[1]    80883 segmentation fault  python3 utils/cora.py --dataset=Cora

**Comment by rusty1s** [comment]
Your installation of `torch-scatter` failed. Can you re-install it and see whether an error occurred?
